scenarios/reproducers/va-nfv-ovs-dpdk-sriov.yml

```diff
@@ -0,0 +1,156 @@
+---
+# This is local to your desktop/laptop.
+# We can't use ansible_user_dir here, unless you have the same user on the
```

Comment:
This is a bit confusing; "We can't use ansible_user_dir" and the we proceed to do so.
iiuc, what this is trying to say is that "We can only use ansible_user_dir if you have the same user on the hypervisor and locally - otherwise edit this?

Reply:
Done, I just copypasted what we had in va-hci.